cpp/include/cudf/strings/detail/gather.cuh

```diff
@@ -68,40 +65,56 @@ __global__ void gather_chars_fn_string_parallel(StringIterator strings_begin,
                                                 MapIterator string_indices,
                                                 size_type total_out_strings)
 {
+  constexpr size_t datatype_size    = sizeof(uint4);
+  constexpr size_t threads_per_warp = 32;
```

Comment:
We already have `cudf::detail::warp_size` (see include/cudf/detail/utilities/cuda.cuh)

Reply:
Implemented in https://github.com/rapidsai/cudf/pull/7980/commits/da0b5254e3eccfc9aed0ad2803e1ba19d522c9ee. Thanks for the suggestion.